index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css\style.css"/>
+<body>
+ <div id="header"><h1 id="title">Microverse Survey Form</h1>
+  <p id="description">It's all because of you that we got this far.Thank you!</p>
+ </div>
+  <form id="survey-form">
+   <label id="name-label">Name</label>
+   <br>
+   <input id="name" type="text" placeholder="Enter your name" required>
+   <br>
+   <label id="email-label" >Email</label>
+   <br>
+   <input id="email" type="email" placeholder="Enter your email" required>
+   <br>
+   <label id="number-label">Age</label>
+   <br>
+     <input id="number" type="number" placeholder="Enter your age" min=15 max=125 >
+     <br>
+     <label>What's your occupation</label>
+     <br>
+     <select id="dropdown">
+       <option> Choose from below</option> 
+       <option>Software Engineer</option>
+       <option>Teacher</option>
+       <option>Banker</option>
+       <option>Student</option>
+     </select>
+     <br>
+     <label>Are you willing to work with Microverse</label>
+     <br>
+     <input value="Absolutely" name="decision" type="radio"> 
+     <label>Absolutele</label>
+     <br>
+     <input value="Mayb" name="decision" type="radio">
+     <label>Maybe</label>
+     <br>
+     <input value="Not sure" name="decision" type="radio">
+     <label>Not sure</label>
+     <br>
+     <legend>what pragramming languages do you prefer(choose all that apply)</legend>
+     <br>
+     <input value="javascript" type="Checkbox">
+     <label>javascript</label>
+     <br>
+     <input value="python" type="checkbox">
+     <label>python</label>
+     <br>
+      <input value="C++" type="checkbox">
+      <label>C++</label>
+     <br>
+        <input value="React" type="checkbox">
+        <label>React</label>
+     <br>
+       <input value="Angular" type="checkbox">
+       <label>Angular</label>
+     <br>
+       <input value="php" type="checkbox">
+       <label>php </label>
+     <br>
+       <input value="swift" type="checkbox">
+       <label>swift</label>
+     <br>
+       <input value="visual basic" type="checkbox">
+       <label>visual basic</label>
+     <br>
+     <div id="bottom">
+     <textarea placeholder="Add comment"></textarea>
```

Comment:
![Screenshot 2023-07-10 093900](https://github.com/Kwamay/Review-Project/assets/69603174/aa407317-1379-4d73-b2de-f4b2d840b063)

I suggest that you bring the "add comment" in the text area to the center by using text-align or add padding in the corresponding CSS file. The current location of the "add comment" doesn't make it look professional since it is closer to the border radius.